src/common/errors/errors-messages.ts

```diff
@@ -0,0 +1,18 @@
+export const ERROR_MESSAGES = {
+  auth: {
+    notAuthorized: {
+      message: 'not authorized',
+    },
+    invalidMessage: {
+      message: 'invalid message',
+    },
+    forbidden: {
+      message: 'forbidden resource',
+    },
+  },
+  server: {
```

Comment:
don't need to duplicate default error, You should to create a data structure and put it to swagger response when server throw Error